addons/l10n_in_ewaybill_irn/models/l10n_in_ewaybill.py

```diff
@@ -27,6 +27,17 @@ def _compute_is_process_through_irn(self):
                 )
             )
 
+    def _check_configuration(self):
```

Comment:
Why not reuse the _check_state?  (you could even give the method a clearer name)